tmt/utils.py

```diff
@@ -1079,6 +1113,26 @@ def default_branch(repository, remote='origin'):
         return ref.read().strip().split('/')[-1]
 
 
+def parse_dotenv(content: str) -> Dict[str, str]:
+    """ Parse dotenv (shell) format of variables """
+    try:
+        return dict([line.split("=")
```

Comment:
Fails if variable contains `=` inside (e.g. passwords, base64 strings..) 
For example:
```
Failed to extract variables from:
SECRET="YQo="
```

Reply:
Good catch!

Reply:
Covered in af814e6.